### DISCUSSION TITLE
Cannot load Driver Script after Schema Build

### DISCUSSION THREAD

**Discussion by LingfuZ** [TPC-Council/HammerDB#173] [Q&A] [answered]
![schema complete 2020-12-01 101002](https://user-images.githubusercontent.com/925839/100779781-bac15500-33bd-11eb-860e-3bbdc38accc7.png)

I'm able to connect with local SQL server and build schema but every time after schema complete, I cannot open Driver Script Options, even not able to open any options in the left hand Benchmark panel. The following is the script from the script editor.
[hammerdb_build_script.zip](https://github.com/TPC-Council/HammerDB/files/5624582/hammerdb_build_script.zip)

HammerDB version is 4.0
Here is the local SQL server info:
![image](https://user-images.githubusercontent.com/925839/100781256-a716ee00-33bf-11eb-9647-43de68e9cf96.png)

And here is the local SQL server connection setting:
![image](https://user-images.githubusercontent.com/925839/100780719-ebee5500-33be-11eb-8437-a6471b1b32ae.png)

Let me know if there are any information you need, I'm currently stuck at Driver Script loading step.

**Comment by sm-shaw** [comment, marked as answer]
You can access all of the documentation here https://www.hammerdb.com/document.html.
If you have not used HammerDB before then the Quick Start section is a good place to start to learn the typical workflow. 
Right now you are here after the schema build has completed https://www.hammerdb.com/docs/ch02s01.html (see Figure 2.5).
Here it says: 

**"Wait until the schema build completes and then click on the red button, tagged with Destroy Virtual Users."**

All of the options are greyed out in the treeview when a workload is running. Therefore you need to click the Red button once complete. This will then reactivate the treeview and you can move on to configuring and running the driver script.

**Comment by LingfuZ** [comment]
Thanks @sm-shaw ! I missed that instruction and the issue is resolved now.
While I have your attention, I'm wondering if I can use HammerDB to run load test with custom defined queries on an existing database?

**Comment by sm-shaw** [comment]
Yes of course.  Use the existing driver script as a template of how to interact with your database and then change the workload and queries as you wish. You can save and load your custom files from the File menu.

**Comment by sm-shaw** [comment]
Closed as resolved.
